### PR TITLE
Scally Core -> Base

### PR DESCRIPTION
**Pull request by chris-pearce**

### CODE REVIEW

File: core/_settings.scss [hunk 1]
```diff
@@ -116,7 +129,7 @@ $col-classes-at-breakpoint-desk-large: false !default;
  */
 
 // Base
-$font-size: 14 !default;
+$font-size: 16 !default;
```
Comment (kllevin) [inline core/_settings.scss]: :+1:

File: core/_settings.scss [hunk 2]
```diff
@@ -346,37 +359,27 @@ $color-black: #000 !default;
...
  */
 
-// Dark
-$color-grey-dark: #002b36 !default;
-
-// Medium
-$color-grey-medium: #586e75 !default;
-
-// Light
-$color-grey-light: #839496 !default;
```
Comment (kllevin) [inline core/_settings.scss:359]: Is all of this moving to Westfield Core?

File: core/base/_forms.scss [hunk 3]
```diff
@@ -49,85 +69,45 @@ textarea[disabled],
...
-  */
+@if $form-text-input-and-select-styles {
 
-  .txt-input,
+  .text-input,
   textarea,
   select {
-    color: $color-txt-base;
+    @include to-rem(padding, $spacing-quarter);
+    color: $color-form-input-txt;
```
Comment (kllevin) [inline core/base/_forms.scss]: change to $color-form-input-text for consistency

File: core/base/_headings.scss [hunk 4]
```diff
@@ -8,20 +8,22 @@
...
-h4, .h4,
-h5, .h5,
-h6, .h6 {
-  @include to-rem(margin-bottom, $spacing-half);
+  // Base
+  h1, .h1,
+  h2, .h2,
+  h3, .h3 {@extend %core-bottom-spacing;}
 
-  &:last-child {margin-bottom: 0;}
```
Comment (kllevin) [inline core/base/_headings.scss:23]: why did you remove this?

File: core/base/_print.scss [hunk 5]
```diff
@@ -28,7 +28,7 @@
 
 
 	/**
-	 * Hide elements not really useful on a peice of paper.
+	 * Hide elements not really useful on a peice of paper
```
Comment (kllevin) [inline core/base/_print.scss]: piece :)

File: core/_settings.scss (hunk 2 above)
Comment (chris-pearce) [inline core/_settings.scss:359]: Yeah, the full colour palette will live there. Don't want to specify to many colours in Scally.

File: core/base/_forms.scss (hunk 3 above)
Comment (chris-pearce) [inline core/base/_forms.scss]: Good catch :+1: 

BTW there's this thing called markdown: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet ;)

File: core/base/_headings.scss (hunk 4 above)
Comment (chris-pearce) [inline core/base/_headings.scss:23]: Cause it exists in the `%core-bottom-spacing;` silent placeholder selector. See: https://github.com/westfieldlabs/scally/blob/base/core/placeholders/_core-bottom-spacing.scss.

File: core/base/_print.scss (hunk 5 above)
Comment (chris-pearce) [inline core/base/_print.scss]: Nice catch :+1:

File: core/_reset.scss [hunk 6]
```diff
@@ -102,11 +102,13 @@ main {
  */
 
 @if $friendly-box-model == true {
```
Comment (AlecRaeside) [inline core/_reset.scss]: If the variable is a Boolean, the `== true` is optional, so it can be written as `@if $friendly-box-model {`

File: core/_reset.scss [hunk 7]
```diff
@@ -102,11 +102,13 @@ main {
...
+
   * {
     &,
     &:before,
     &:after {box-sizing: border-box;}
   }
+
 }/* endif */
```
Comment (AlecRaeside) [inline core/_reset.scss]: Are we switching to `//` comments?

File: core/_settings.scss [hunk 8]
```diff
@@ -346,37 +359,27 @@ $color-black: #000 !default;
...
- * Buttons
+ * Buttons / CTA's
  */
 
 // Primary
-$color-btn-primary: $color-brand !default;
+$color-btn-cta-primary: $color-brand !default;
 
 // Secondary
-$color-btn-secondary: #cb4b16 !default;
```
Comment (AlecRaeside) [inline core/_settings.scss:379]: For the groups of variables which have a common prefix on them, we should use sass maps http://viget.com/extend/sass-maps-are-awesome . This will make generating helpers for each colour easier. But using them is slightly harder as you need to use the map-get function to get a value. Will need some more investigation before implementing

File: core/base/_media.scss [hunk 9]
```diff
@@ -21,14 +23,13 @@ video {vertical-align: middle;}
...
-  // Images in `figure` elements
-  figure > & {display: block;}
+	max-width: 100%; // [1]
+	height: auto; // [1]
+	font-style: italic; // [2]
 
-  // Non-responsive images if you specify `width` and/or `height` attributes
+  // Non-responsive images if you specify `width` and/or `height` attributes and Google Maps breaks
+  // if `max-width: 100%` acts upon it; use their selector to remove the effects.
+  .gm-style &,
```
Comment (AlecRaeside) [inline core/base/_media.scss:36]: Just thinking out loud here, should third-party overrides live in a separate file?

**Comment by AlecRaeside** [comment]
Some minor comments, otherwise LGTM. Sass maps may be a good option for the colours variables etc, I'll look into it.

File: core/_reset.scss (hunk 6 above)
Comment (chris-pearce) [inline core/_reset.scss]: Yep I changed em all in **Base**, just changed this one too. I'll be doing a global search & replace in another PR.

File: core/_reset.scss (hunk 7 above)
Comment (chris-pearce) [inline core/_reset.scss]: Yes, I've changed this. Again will be doing a global search & replace in a separate PR.

File: core/_settings.scss (hunk 8 above)
Comment (chris-pearce) [inline core/_settings.scss:379]: Yes I definitely want to use Sass maps where appropriate and seems in this case it is? Will look at it in a separate PR.

File: core/base/_media.scss (hunk 9 above)
Comment (chris-pearce) [inline core/base/_media.scss:36]: This is a funny one, it went in yesterday as I saw it in [Inuit](https://github.com/inuitcss/base.images/blob/master/_base.images.scss#L26) and thought it's a good addition as I've been caught out by that many times before. Yes maybe it can live in a separate `vendor` partial? Will add it to the `TODO.md` for discussion.
